file-mon/src/main/java/com/wipro/ats/bdre/filemon/QueuedFileUploader.java

```diff
@@ -94,12 +95,9 @@ private static void executeRegisterFiles(FileCopyInfo fileCopying) {
             long timeStamp = fileCopying.getTimeStamp();
             Date dt = new Date(timeStamp);
             String strDate = sdf.format(dt);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)